gc/stats/ScavengerHotFieldStats.hpp

```diff
@@ -295,6 +297,18 @@ class MM_ScavengerHotFieldStats : public MM_Base
 		omrtty_printf(" }\n");
 		omrtty_printf("{ Hot Field Statistics nursery-tenured: end }\n");
 	}
+	
+	/**
+	 * Create a SvavengerHotFieldStats object.
+	 */
+
+	MM_ScavengerHotFieldStats()
+		: _objectModel(NULL)

```

Comment:
please put ":" at the end of previous line

Reply:
👍